depenguinme.sh

```diff
@@ -58,7 +58,7 @@ usage() {
 }
 
 is_url() {
-	[[ "$1" =~ ^(http|https|ftp):// ]]
+	[[ "$1" =~ ^(http|https|ftp|sftp):// ]]
```

Comment:
Is sftp supported by wget?

Reply:
"unsupported scheme 'sftp'" on my test box

Reply:
Yes, not supported :(